fedoratagger/lib/retired.py

```diff
@@ -23,49 +23,55 @@
 log.setLevel(logging.DEBUG)
 logging.basicConfig()
 
-PKGDB_URL = 'https://admin.fedoraproject.org/pkgdb/api/packages'
+PDC_URL = 'https://pdc.fedoraproject.org/rest_api/v1/component-branches'
 
-def get_packages(status):
-    log.info('Retrieve the list of %s packages.' % status)
-    total = 1
+
+def get_packages(is_active=False):
+    log.info('Retrieve the list of packages.')
     page = 1
+    next_page = 1
     packages = []
-    while page <= total:
+    while next_page:
         args = {
-            'status': status,
+            'active': is_active,
             'page': page,
```

Comment:
I think we should only looked at the current release package, so I would had the following argument `'name': 'f28'`